app/Enum/CacheKey.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Enum;
```

Comment:
I'm not really happy with the location of that file, but I cannot find a better location. I cannot use the PHP `enum` data type since there's a function, too.